### PR TITLE
Link TFOptimizer to tfpark

### PR DESCRIPTION
**Pull request by Bob-Chou**

### CODE REVIEW

File: pyzoo/zoo/tfpark/__init__.py [hunk 1]
```diff
@@ -15,4 +15,5 @@
 #
 
 from zoo.pipeline.api.net import TFDataset
+from zoo.pipeline.api.net import TFOptimizer
```
Comment (hkvision) [inline pyzoo/zoo/tfpark/__init__.py]: combine with TFDataSet into one line :)

**Comment by Bob-Chou** [comment]
Jenkins passed: http://10.239.47.211:18888/job/ZOO-PR-Validation/2431/console

**Comment by yangw1234** [comment]
LGTM

**Comment by jason-dai** [comment]
We also need to update the document

**Comment by hkvision** [comment]
@Bob-Chou Let's update the document and README as well.

**Comment by Bob-Chou** [comment]
docs update pls refer to https://github.com/intel-analytics/analytics-zoo/pull/1515

**Pull request merged**